app/src/main/java/com/rockthevote/grommet/ui/views/AddressView.java

```diff
@@ -57,27 +60,31 @@
 
     @Inject Moshi moshi;
 
+    @Pattern(regex = ValidationRegex.ADDRESS)
```

Comment:
move all `NotEmpty` validations above the other ones when there's more than one.

Also need to add a message for when the new pattern fails

Finally, I do not think the regex is valid and have pinged Alex on this